### DISCUSSION TITLE
Power Fx assemblies in Nuget vs what comes with pac

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/Power-Fx#2372] [General]
I am trying to use Power Fx in Polyglot Notebooks to query Dataverse and I always get an error with engine Check when I use Power Fx 1.2.0 assemblies. When I investigated how pac REPL is able to do this without any error, I found that it uses different version that is not even published on Nuget.

![image](https://github.com/microsoft/Power-Fx/assets/5035266/18083262-8689-4dc6-b942-3b22898a43ec)

[nuget](https://www.nuget.org/packages?q=Microsoft.PowerFx) does not have version higher than 1.2.0, but all Power Fx assemblies referred by pac are on 1.3.0.

Ideally pac should also only use what is published on nuget. I currently have to downgrade to 1.1.0 to get the Dataverse Power Fx part working.

**Comment by rajyraman** [comment]
Switched to [nightly](https://github.com/microsoft/Power-Fx/blob/main/dailyBuilds.md)

**Comment by MikeStall** [comment]
Yes, use nightly. WE only publish to nuget every few months. Nightly are produced on every commit. 
However, we are having an interruption in publishing nightly builds, tracked with: #2344. 

Fwiw - for querying dataverse, check out: 
https://github.com/microsoft/power-fx-host-samples/blob/main/Samples/Dataverse101/Program.cs

**Comment by rajyraman** [comment]
@MikeStall - Intellisense was not working for Dataverse, but I realised that I did not populate the SymbolTable from DataverseConnection. Got it working today.

All the changes are in my repo now https://github.com/rajyraman/Power-Fx-.NET-Interactive

Thank you for your help Mike.

![image](https://github.com/microsoft/Power-Fx/assets/5035266/4fe203e6-4974-4245-84f2-7a31cb9add54)
